File: app/src/main/java/com/example/client/screens/register/present/IRegisterPresent.java
package com.example.client.screens.register.present;

public interface IRegisterPresent {
    void onRegister(String fullname,String phone, String email, String password);
    void onGetUserActive(String email);
    void onUpdateDeviceToken(String email, String token);
    void sendEmail(String email);
    void verification(String email, String code, String fullname, String phone, String password);
}

File: app/src/main/java/com/example/client/screens/register/present/IRegisterPresent.kt
package com.example.client.screens.register.present

import com.example.client.base.IBasePresenter

interface IRegisterPresent: IBasePresenter {
    fun register(fullname: String, phone: String, email: String, password: String)
    fun setUserActive(email: String)
    fun updateDeviceToken(email: String, token: String)
    fun sendEmail(email: String, phone: String)
    fun verification(email: String, code: String, fullname: String, phone: String, password: String)
}